frontend/src/utils/blockUtils.js

```diff
@@ -26,11 +26,19 @@ export const genJSON = (block, id) => {
   };
 };
 
-export function generateId(block) {
-  const nanoid = customAlphabet("1234567890abcedfghijklmnopqrstuvwxyz", 12);
+export function generateId(entity, prefix) {
```

Comment:
I may be wrong, but I think we could get away with only passing the prefix here?

Reply:
yeah. thanks for pointing this out. I have changed it.